ogre/src/media/materials/scripts/lidar_visual.material

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+material Lidar/LightBlueLaser
```

Comment:
The material script is specific to ogre 1.x which won't be used by other rendering engine implementations. So you may want to create materials manually by hand, [see examples here](https://github.com/ignitionrobotics/ign-rendering/blob/master/src/base/BaseScene.cc#L1150). You could create them in BaseLidarVisual so that when you work on ogre2 implementation, these materials will be available.